Gemfile

```diff
@@ -3,10 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
+gem 'rspec'
```

Comment:
Do we need rspec running on production or rather we are going to use it in testing/development environment? So I would move it to testing/dev section. Also please change `rspec` -> `rspec-rails`